stdlib/messaging/kafka/src/main/ballerina/src/kafka/kafka_constants.bal

```diff
@@ -100,3 +103,15 @@ public const COMPRESSION_LZ4 = "lz4";
 
 # Kafka ZSTD compression type.
 public const COMPRESSION_ZSTD = "zstd";
+
+// ********************************************
+//              Common constants              *
+// ********************************************
+// SASL Authentication mechanisms
+# Kafka SASL_PLAIN authentication mechanism
+public const AUTH_SASL_PLAIN = "PLAIN";
+
+
+// Security Protocols
```

Comment:
```suggestion
// Security Protocols.
```

Reply:
This isn'n an API documentation comment.